### PR TITLE
ZH-trans: update images.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ content_template: templates/concept
 {{% capture body %}}
 
 ## 升级镜像
-默认的镜像拉取策略是“IfNotPresent”，在镜像已经存在的情况下，kubelet将不在去拉取镜像。
-如果总是想要拉取镜像，必须设置拉取策略为“Always”或者设置镜像标签为“:latest”。
+默认的镜像拉取策略是 `IfNotPresent`，在镜像已经存在的情况下，kubelet将不在去拉取镜像。
```
Comment (idealhack) [inline content/zh/docs/concepts/containers/images.md]:
```suggestion
默认的镜像拉取策略是 `IfNotPresent`，在镜像已经存在的情况下，Kubelet 将不再拉取镜像。
```

**Comment by xichengliudui** [comment]
好的，我查看一下进行更新。

**Comment by xichengliudui** [comment]
后续更新。

**Pull request closed without merging**